test/transaction/sidechain-certificate.js

```diff
@@ -38,37 +38,35 @@ describe('#Sidechain certificate', function() {
         //scProof
         assert.equal(cert.scProof, certJson.cert.scProof);
         //vFieldElementCertificateField
-        assert.deepEqual(cert.vFieldElementCertificateField,certJson.cert.vFieldElementCertificateField);
+        assert.deepEqual(cert.vFieldElementCertificateField, certJson.cert.vFieldElementCertificateField);
         //vBitVectorCertificateField
-        assert.deepEqual(cert.vBitVectorCertificateField,certJson.cert.vBitVectorCertificateField);
+        assert.deepEqual(cert.vBitVectorCertificateField, certJson.cert.vBitVectorCertificateField);
         //ftScFee
         assert.equal(cert.ftScFee / 1e8, certJson.cert.ftScFee);
         //mbtrScFee
         assert.equal(cert.mbtrScFee / 1e8, certJson.cert.mbtrScFee);
 
         //vout
-        assert.equal(cert.outputs.length,certJson.vout.length);
-        for (var i=0; i<certJson.vout.length;i++) {
-          assert.equal(cert.outputs[i].toObject().satoshis,certJson.vout[i].valueZat);
-          assert.equal(cert.outputs[i].toObject().script,certJson.vout[i].scriptPubKey.hex);
-          if (cert.outputs[i].toObject().isFromBackwardTransfer && cert.outputs[i].toObject().isFromBackwardTransfer == true) {
-            assert.equal(cert.outputs[i].toObject().isFromBackwardTransfer,certJson.vout[i]["backward transfer"]);
-            assert.equal(BufferUtil.reverse(Buffer.from(cert.outputs[i].toObject().pubKeyHash, 'hex')).toString('hex'),certJson.vout[i].pubkeyhash);
+        assert.equal(cert.outputs.length, certJson.vout.length);
+        for (let i = 0; i < certJson.vout.length; i++) {
+          assert.equal(cert.outputs[i].toObject().satoshis, certJson.vout[i].valueZat);
+          assert.equal(cert.outputs[i].toObject().script, certJson.vout[i].scriptPubKey.hex);
+          if (cert.outputs[i].toObject().isFromBackwardTransfer &&
+            cert.outputs[i].toObject().isFromBackwardTransfer === true) {
+            assert.equal(cert.outputs[i].toObject().isFromBackwardTransfer, certJson.vout[i]['backward transfer']);
```

Comment:
With the new zendoo this field will be rename to "backwardTransfer".

Reply:
only affects the tests